.pre-commit-config.yaml

```
@@ -61,6 +62,11 @@ repos:
       - id: yamllint
         args: [-f=standard, -c=.yamllint.yml]
         files: src/schema/.*/.*\.yaml
+  - repo: https://github.com/macisamuele/language-formatters-pre-commit-hooks
+    rev: v2.14.0
+    hooks:
+      - id: pretty-format-toml
+        args: [--autofix, --indent, '4']
```

I don't think sorting leads to a very readable `pyproject.toml`, where I think there are some rough importance levels that are good to indicate (e.g. "name" should be the first thing in project metadata).

```suggestion
        args: [--autofix, --indent, '4', --no-sort]
```

---

good with me
defo not a hill I want to die on

---

forgot to commit this one